variables.tf

```
@@ -69,6 +69,18 @@ variable "firewall_rules" {
   }
 }
 
+variable "server_security_alert_policy_email_addresses" {
+  description = "A list of custom email addresses to which the alert for this SQL server is sent."
+  type        = list(string)
+  default     = []
+}
+
+variable "server_security_alert_policy_email_account_admins" {
+  description = "Is alerts for this SQL server sent to account adminitrators?"
```

Clarify who receives alerts:

```suggestion
  description = "Are alerts for this SQL server sent to subscription owners and administrators?"
```